package.json

```diff
@@ -456,6 +477,11 @@
       "type": "object",
       "title": "PowerShell Configuration",
       "properties": {
+        "powershell.sideBar.CommandExplorerVisibility": {
+          "type": "boolean",
+          "default":true,
```

Comment:
There should've been a space after the colon (`:`). I opened PR #1688 to fix it. Running `Invoke-Build` at the end helps with that.